recipe/meta.yaml

```diff
@@ -11,6 +11,7 @@ source:
   sha256: 4f297f285b4d9d9f8dd31d3944c6b15073883b6416e6ac12802f9c6d1c6642fa
 
 build:
+  number: 1
```

Comment:
You added the number here instead of bumping the `number: 0` below

Reply:
Indeed! 

(shouldn't listen to talks and do things on a side - I even checked what the number is if not given, but didn't see it that it was way below in the file)